framework/core/Vcs.pm

```diff
@@ -288,7 +289,9 @@ sub checkout_vid {
 
     # Get and run specific checkout command
     my $cmd = $self->_checkout_cmd($revision_id, $work_dir);
-    Utils::exec_cmd($cmd, "Checking out " . _trunc_rev_id($revision_id) . " to $work_dir") or return 0;
+    Utils::exec_cmd($cmd, "Checking out revision " . _trunc_rev_id($revision_id) . " to $work_dir") or return 0;
+    print(STDERR strftime "%Y-%m-%d %H:%M:%S", gmtime time);
+    print(STDERR "\n");
```

Comment:
Should we move this to `Utils::exec_cmd` and guard it with a check on the [DEBUG](https://github.com/rjust/defects4j/blob/master/framework/core/Constants.pm#L51) flag? It might make sense to generally run D4J in debug mode on travis.

Reply:
I made this change.  I agree with the comment about debug mode on Travis.  I think that should go in a different pull request.